src/ZendCodingStandard/Sniffs/Files/CopyingSniff.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-coding-standard for the canonical source repository
+ * @copyright https://github.com/zendframework/zend-coding-standard/blob/master/COPYING.md Copyright
+ * @license   https://github.com/zendframework/zend-coding-standard/blob/master/LICENSE.md New BSD License
+ */
+
+use PHP_CodeSniffer_File as File;
+use PHP_CodeSniffer_Sniff as Sniff;
+use Zend\CodingStandard\Utils\LicenseUtils;
+
+/**
+ * COPYING.md Sniff
+ *
+ * - Checks and creates COPYING.md in the project root dir
+ * - Checks and fixes copyright in COPYING.md; it should be the current year
+ */
+class ZendCodingStandard_Sniffs_Files_CopyingSniff implements Sniff
+{
+    /**
+     * @var SplFileInfo
+     */
+    private $copyrightFile;
+
+    public function __construct()
+    {
+        $this->copyrightFile = LicenseUtils::getCopyrightFile();
+    }
+
+    /**
+     * Registers the tokens that this sniff wants to listen for.
+     *
+     * @return int[]
+     */
+    public function register()
+    {
+        return [T_OPEN_TAG];
+    }
+
+    /**
+     * Called when one of the token types that this sniff is listening for is
+     * found.
+     *
+     * @param File $phpcsFile The PHP_CodeSniffer file where the token was found.
+     * @param int $stackPtr The position in the PHP_CodeSniffer file's token
+     *                      stack where the token was found.
+     *
+     * @return int Optionally returns a stack pointer. The sniff will not be
+     *             called again on the current file until the returned stack
+     *             pointer is reached.
+     */
+    public function process(\PHP_CodeSniffer_File $phpcsFile, $stackPtr)
```

Comment:
Shouldn't the typehint be `File` here? (You alias that to this class in the import statements...)